### ISSUE TITLE
Fix documentation around new note references

### ISSUE THREAD

**Issue by ScriptAutomate** [dendronhq/dendron-site#65]
This is an extension of the conversation in the `dendronhq/dendron` repo:
- https://github.com/dendronhq/dendron/issues/530

## Docs Problem

I've been trying to figure out the new way of using reference notes, which was mentioned in a recent PR of mine (https://github.com/dendronhq/dendron-site/pull/60), as I was still using the old way. Detailed here is the newer way:
- https://dendron.so/notes/f1af56bb-db27-47ae-8406-61a98de6c78c.html#references

I didn't realize there was some cool new ways of working with note references via `![[foo]]` instead of `((ref: [[foo]]))`, but I am struggling with the actual implementation.

This format gets rendered as plain text, and isn't picked up by Dendron at build at all:

```
![[ dendron.community.showcase
    #Open AWS Catalogue,1
    :#*
]]
```

Trying to condense together at least gets rendered, but doesn't work as expected:

```
![[dendron.community.showcase#Open AWS Catalogue,1:#*]]
```

I get the on-page rendered error of: `Start anchor Open AWS Catalogue not found` (but it does exist!)

The old format still works:

```
((ref:[[dendron.community.showcase]]#Open AWS Catalogue,1:#*]]))
```

After playing around, I realized the new format works **only when there is no whitespace in a header reference, and the whitespace is replaced with a `-`**. This means that this rendered properly for the `Open AWS Catalogue` header:

```
![[dendron.community.showcase#Open-AWS-Catalogue,1:#*]]
```

## Docs Fix

@kevinslin said the following in https://github.com/dendronhq/dendron/issues/530#issuecomment-796368140:

>  we need to update the docs. the new style note refs use github style slugging
> this is because old note refs weren't able to handle special characters in the headers
> tldr: whitespace is no longer supported because github-slugger doesn't support it. if you use the `cmd+shift+r` shortcut on a header, we will automatically sluggify the note ref

**Comment by ScriptAutomate** [comment]
Side note: There is also the collection of old note ref formats.

List of `grep` findings:
```
20:12 $ grep -rn '((ref:' | egrep -v 'build/|Binary file|node_modules/'vault/dendron.topic.extensions.md:30:((ref:[[dendron.guides.tips]]#other VSCode extensions:#other Browser Extensions))
vault/dendron.topic.multi-vault.md:34:((ref:[[dendron.topic.commands]]#vault add:#*))
vault/dendron.topic.multi-vault.md:36:((ref:[[dendron.topic.commands]]#vault remove:#*))
vault/dendron.topic.lookup.modifiers.md:57:((ref:[[dendron.topic.config]]#dendron.linkselectautotitlebehavior,1:#*))
vault/dendron.concepts.md:61:((ref:[[dendron.topic.hierarchies]]#Hierarchies,1:#Concepts))
vault/dendron.topic.schema.md:261:((ref:[[dendron.topic.schema]]#parent:#*))
vault/demo.refs.wildcard.md:9:((ref:[[demo.refs.wildcard.target]]#h1.1:#*))
vault/demo.refs.wildcard.md:13:((ref:[[demo.refs.wildcard.target]]#h1.2,1:#*))
vault/dendron.release.2020-12-28.md:55:((ref:[[dendron.release.changelog]]#0.21.2:#0.20.2))
vault/dendron.pro.markdown-links.setup.md:31:   ((ref:[[dendron.topic.cli]]#launchengineserver,1:#*))
vault/dendron.topic.lookup.keybindings.md:10:((ref:[[dendron.topic.keybindings]]#keybindings,1))
vault/dendron.topic.publishing-legacy.commands.md:10:((ref:[[dendron.topic.commands]]#publishing,1:#Dev))
vault/dendron.release.changelog.md:346:You can now do references to notes using `![[note]]` instead of `((ref: [[note]]))` syntax. To reference a header, use `![[note#foo]]`. This changes makes our note ref syntax more consistent with our wiki link syntax. 
vault/dendron.release.2020-11-29.md:51:((ref:[[dendron.release.changelog]]#0.17.2:#0.16.3))
vault/dendron.workflows.todos.md:61:((ref:[[dendron.guides.tips]]#Moving lines,1:#*))
vault/dendron.release.2020-11-22.md:69:((ref:[[dendron.release.changelog]]#0.16.3:#0.16.1))
vault/dendron.topic.tags.md:32:((ref:[[dendron.guides.tips]]#stylized tags using custom css:#*))
vault/dendron.release.2020-12-21.md:50:((ref:[[dendron.release.changelog]]#0.20.2:#0.19.3))
vault/dendron.topic.frontmatter.md:67:((ref:[[dendron.topic.publishing.configuration]]#Note Configuration))
vault/dendron.roadmap.project.n.2020.multi-vault.md:46:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#file layout for legacy workspace,1:#*))
vault/dendron.roadmap.project.n.2020.multi-vault.md:50:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#file layout for multi-vault workspace,1:#*))
vault/dendron.roadmap.project.n.2020.multi-vault.md:88:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#file layout for multi-vault workspace,1:#*))
vault/dendron.roadmap.project.n.2020.multi-vault.md:123:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#file layout for multi-vault workspace,1:#*))
vault/dendron.roadmap.project.n.2020.multi-vault.md:139:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#file layout for multi-vault workspace,1:#*))
vault/dendron.release.2020-09-13.md:19:((ref:[[dendron.topic.refs]]# references,1:# *)) 
vault/dendron.release.2020-09-13.md:33:((ref:[[dendron.topic.cli]]#install))
vault/dendron.release.2020-09-13.md:55:((ref:[[dendron.topic.commands]]#configure pod,1:#*))
vault/dendron.release.2020-09-13.md:71:((ref:[[dendron.topic.publishing.configuration]]#useprettyrefs,1:#*))
vault/dendron.release.2020-09-13.md:75:((ref:[[dendron.topic.publishing.configuration]]#noindex,1:#*))
vault/dendron.release.2020-09-13.md:90:((ref:[[dendron.topic.refs]]#wildcard header reference,1:#*))
vault/dendron.release.2020-09-13.md:105:((ref:[[dendron.pro.dendron-snippet-maker]]#dendron snippet maker,1))
vault/dendron.workflows.cache.md:34:((ref:[[dendron.topic.publishing.advanced]]#exclude from indexing,1:#*))
vault/dendron.topic.pod.builtin.json.md:170:    "body": "\n\n# Inner Note\n\n((ref: [[bar.two.child]]))\n((ref: [[bar.two.child]]))",
vault/dendron.release.2020-11-15.md:55:((ref:[[dendron.release.changelog]]#0.15.3:#0.14.2))
vault/dendron.release.2020-12-14.md:58:((ref:[[dendron.release.changelog]]#0.19.3:#0.18.2))
vault/dendron.dev.md:14:((ref:[[dendron.dev.setup]]#getting started))
vault/dendron.release.2021-01-04.md:61:((ref:[[dendron.release.changelog]]#0.22.2:#0.21.2))
vault/dendron.release.2020-12-07.md:52:((ref:[[dendron.release.changelog]]#0.18.2:#0.17.2))
vault/dendron.release.2020-11-08.md:24:((ref:[[dendron.community.roles]]#dendrologist,1:#*))
vault/dendron.release.2020-11-08.md:48:((ref:[[dendron.release.changelog]]#changelog,1:#13.6))
vault/dendron.dev.api.seeds.md:9:((ref:[[dendron.topic.seeds]]#seeds 🚧,1:#*))
vault/dendron.topic.hierarchies.md:43:((ref:[[dendron.ref.layout]]#File Tree,1:#Additional Folders))
vault/dendron.dev.design.engine.md:279:((ref:[[dendron.topic.refs]]#references,1:#*))
vault/dendron.topic.pod.markdown.md:65:((ref: [[pro.dendron.feat.publish]]))
vault/dendron.topic.pod.markdown.md:129:((ref:[[dendron.topic.refs]]#references,1:#*))
vault/dendron.guides.cook.md:172:((ref:[[dendron.topic.commands]]#initialize workspace:#*))
vault/dendron.guides.cook.md:226:  ((ref:[[dendron.topic.workbench]]#outline view,1:#*))
vault/dendron.guides.cook.md:229:  ((ref:[[dendron.topic.preview]]#table of contents,1))
vault/dendron.guides.cook.md:253:  ((ref:[[dendron.topic.cli]]#installation,1:#*))
vault/dendron.workflows.bullet-journal.md:20:((ref:[[dendron.topic.special-notes]]#daily journal,1:#*))
vault/dendron.workflows.bullet-journal.md:24:((ref:[[dendron.topic.commands]]#go next sibling,1:#*))
vault/dendron.workflows.bullet-journal.md:28:((ref:[[dendron.topic.schema]]#template,1:#*))
vault/dendron.workflows.bullet-journal.md:34:((ref:[[dendron.guides.tips]]#moving panes,1))
vault/dendron.topic.capabilities.snapshot.md:14:((ref:[[dendron.topic.commands]]#snapshot vault:#*))
vault/dendron.topic.capabilities.snapshot.md:18:((ref:[[dendron.topic.commands]]#restore vault:#*))
vault/dendron.topic.snippets.md:23:((ref:[[dendron.pro.dendron-snippet-maker]]))
vault/dendron.topic.refs.md:26:- NOTE: Dendron recently switched the note ref syntax for `((ref: [[foo]]))` to `![[foo]]`. The videos have not yet been updated to reflect this change
vault/dendron.release.2020-11-01.md:79:((ref:[[dendron.release.changelog]]#13.6:#0.12.11))
vault/dendron.topic.seeds.md:28:((ref:[[dendron.topic.cli]]#installation,1:#*))
vault/dendron.md:49:((ref:[[dendron.guides.install]]#Getting Started,1:#Install))
vault/dendron.release.2020-09-20.md:43:((ref:[[dendron.topic.seeds]]#seeds 🚧,1:#*))
vault/dendron.release.2020-09-20.md:60:((ref:[[dendron.topic.lookup]]#creating notes with a selection,1:#*))
vault/dendron.release.2020-09-20.md:70:((ref:[[dendron.topic.pod.builtin.json]]))
vault/dendron.release.2020-09-20.md:86:((ref:[[dendron.topic.commands]]#publish,1:#*))
vault/dendron.release.2020-09-20.md:90:((ref:[[dendron.topic.commands]]#copy note url,1:#*))
vault/dendron.release.2020-10-19.md:43:((ref:[[dendron.community.showcase]]#open pkm catalogue,1:#*))
vault/dendron.release.2020-10-19.md:44:((ref:[[dendron.community.showcase]]#open aws catalogue,1:#*))
vault/dendron.release.2020-10-19.md:66:((ref:[[dendron.release.changelog]]#0.12.11,1))
vault/dendron.pro.dendron-next-server.md:26:((ref:[[dendron.topic.cli]]#installation,1:#*))
vault/dendron.workflows.academics.md:13:((ref:[[dendron.topic.markdown]]#footnotes,1:#*))
vault/dendron.workflows.academics.md:17:((ref:[[dendron.topic.links]]#other links,1))
vault/dendron.guides.install.md:20:((ref:[[dendron.guides.cook]]#launch the command bar,3:#*))
vault/dendron.release.changelog.0-1-x.md:24:((ref:[[dendron.topic.seed-bank]]))
vault/dendron.release.changelog.0-1-x.md:50:((ref:[[dendron.topic.multi-vault]]#specify vault location when creating a note,1:#*))
vault/dendron.release.changelog.0-1-x.md:143:((ref:[[dendron.topic.commands]]#configure,1:#*))
vault/dendron.release.changelog.0-1-x.md:226:((ref:[[dendron.topic.commands]]# copy note link:# *))
vault/dendron.release.changelog.0-1-x.md:298:((ref:[[dendron.roadmap.project.n.2020.multi-vault]]#use cases,1:#*))
vault/dendron.release.changelog.0-1-x.md:348:((ref:[[dendron.topic.refs]]#wildcard note refs,1))
vault/dendron.release.changelog.0-1-x.md:375:((ref:[[dendron.topic.cli]]#launchengineserver:#*))
vault/dendron.release.changelog.0-1-x.md:395:((ref:[[dendron.topic.lookup.modifiers]]#copywikilink,1:#*))
vault/dendron.release.changelog.0-1-x.md:401:((ref:[[dendron.topic.lookup.modifiers]]#multiselect,1:#*))
vault/dendron.release.changelog.0-1-x.md:457:((ref:[[dendron.topic.commands]]#contribute:#*))
vault/dendron.release.changelog.0-1-x.md:463:((ref:[[dendron.topic.capabilities.snapshot]]#Snapshots:#*))
vault/dendron.release.changelog.0-1-x.md:524:((ref:[[dendron.topic.lookup]]#filter toggle,1:#*))
vault/dendron.release.changelog.0-1-x.md:527:  ((ref:[[dendron.topic.lookup.modifiers]]#filtertype:#*))
vault/dendron.release.changelog.0-1-x.md:532:  ((ref:[[dendron.topic.publishing.configuration]]#config:#*))
vault/dendron.release.changelog.0-1-x.md:536:((ref:[[dendron.topic.schema]]#schema templates,1:#*))
vault/dendron.release.changelog.0-1-x.md:565:((ref:[[dendron.topic.pod]]#publish,1:#*))
vault/dendron.release.changelog.0-1-x.md:592:((ref:[[dendron.topic.pod]]#pods:#*))
vault/dendron.release.changelog.0-1-x.md:602:((ref:[[dendron.topic.publishing.configuration]]#toc:#*))
vault/dendron.release.changelog.0-1-x.md:626:((ref:[[dendron.topic.refs]]#recursive reference,1))
vault/dendron.release.changelog.0-1-x.md:683:((ref:[[dendron.guides.troubleshooting]]#error upgrading:#*))
vault/dendron.release.changelog.0-1-x.md:713:((ref:[[dendron.topic.extensions]]#useful extensions))
vault/dendron.release.changelog.0-1-x.md:715:((ref:[[dendron.guides.tips]]#copy and paste web content into dendron:#*))
vault/dendron.release.changelog.0-1-x.md:717:((ref:[[dendron.guides.cook]]#mobile support:#*))
vault/dendron.release.changelog.0-1-x.md:780:((ref:[[dendron.roadmap.project.n.2020.server-migration]]#tasks,1:#*))
vault/dendron.release.changelog.0-1-x.md:795:((ref:[[dendron.topic.snippets]]#default snippets,1:#*))
vault/dendron.release.changelog.0-1-x.md:799:((ref:[[dendron.pro.dendron-jekyll.topic.math]]#math,1:#*))
vault/dendron.release.changelog.0-1-x.md:803:((ref:[[dendron.topic.publishing.configuration]]#siterepodir,1:#*))
vault/dendron.release.changelog.0-1-x.md:820:  ((ref:[[dendron.roadmap.project.n.2020.server-migration]]#tasks,1:#*))
vault/dendron.release.changelog.0-1-x.md:861:((ref:[[dendron.topic.lookup]]#split toggle))
vault/dendron.release.changelog.0-1-x.md:867:((ref:[[dendron.topic.commands]]#copy note ref:#*))
vault/dendron.release.changelog.0-1-x.md:873:((ref:[[dendron.roadmap]]))
vault/dendron.release.changelog.0-1-x.md:877:  ((ref:[[dendron.topic.lookup]]#lookup menu:#*))
vault/dendron.release.changelog.0-1-x.md:882:  ((ref:[[dendron.community.showcase]]#luke's second brain))
vault/dendron.pro.dendron-jekyll.topic.frontmatter.md:46:((ref:[[dendron.topic.publishing.configuration]]#config:#*))
data/dendron-config.json:116:            "docs": "((ref: [[dendron.topic.lookup]]#notes,1:#schemas))",
data/dendron-config.json:134:            "docs": "((ref: [[dendron.topic.lookup]]#notes,1:#schemas))",
data/dendron-config.json:153:            "docs": "((ref: [[dendron.topic.lookup]]#notes,1:#schemas))",
data/v0.10.7.yml:51:        ((ref: [[dendron.topic.cli]]#install))
data/v0.10.7.yml:84:        ((ref: [[dendron.topic.commands]]#new note from selection,1:#*))
data/v0.10.7.yml:93:        ((ref: [[dendron.topic.commands]]#configure pod,1:#*))
data/v0.10.7.yml:110:        ((ref: [[dendron.topic.pod]]#export config,1))
data/v0.10.7.yml:135:        ((ref: [[dendron.topic.publishing]]#note references,1:#*))
data/v0.10.7.yml:144:        ((ref: [[dendron.topic.publishing]]#exclude from indexing,1:#*))
data/v0.10.7.yml:209:        ((ref: [[dendron.topic.refs]]#wildcard header ref,1))
data/v0.10.7.yml:218:        ((ref: [[dendron.pro.dendron-snippet-maker]]#dendron snippet maker,1))
data/v0.11.9.yml:40:        ((ref: [[dendron.topic.seeds]]#seeds 🚧,1:#*))
data/v0.11.9.yml:65:        ((ref: [[dendron.topic.lookup]]#creating notes with a selection,1:#*))
data/v0.11.9.yml:82:        ((ref: [[dendron.topic.pod]]#json pod,1:#Custom Pods))
data/v0.11.9.yml:123:        ((ref: [[dendron.topic.commands]]#publish,1:#*))
data/v0.11.9.yml:188:        ((ref: [[dendron.topic.commands]]#copy note url,1:#*))
data/v0.9.7.yml:102:        ((ref:[[demo.embed.block]]#head1,1))
```

List of potential example fixes via some terminal `sed` modifications:
```
✔ ~/git/dendron-site [master|✚ 3…2] 
20:12 $ grep -rn '((ref:' | egrep -v 'build/|Binary file|node_modules/' | sed 's:((ref\::!:g' | sed 's:]]::g' | sed 's:)):]]:g' | sed 's:!\ :!:g' | sed 's:\ :-:g' | sed 's:\-\-::g' | sed 's:\-":":g'
vault/dendron.topic.extensions.md:30:![[dendron.guides.tips#other-VSCode-extensions:#other-Browser-Extensions]]
vault/dendron.topic.multi-vault.md:34:![[dendron.topic.commands#vault-add:#*]]
vault/dendron.topic.multi-vault.md:36:![[dendron.topic.commands#vault-remove:#*]]
vault/dendron.topic.lookup.modifiers.md:57:![[dendron.topic.config#dendron.linkselectautotitlebehavior,1:#*]]
vault/dendron.concepts.md:61:![[dendron.topic.hierarchies#Hierarchies,1:#Concepts]]
vault/dendron.topic.schema.md:261:![[dendron.topic.schema#parent:#*]]
vault/demo.refs.wildcard.md:9:![[demo.refs.wildcard.target#h1.1:#*]]
vault/demo.refs.wildcard.md:13:![[demo.refs.wildcard.target#h1.2,1:#*]]
vault/dendron.release.2020-12-28.md:55:![[dendron.release.changelog#0.21.2:#0.20.2]]
vault/dendron.pro.markdown-links.setup.md:31:-![[dendron.topic.cli#launchengineserver,1:#*]]
vault/dendron.topic.lookup.keybindings.md:10:![[dendron.topic.keybindings#keybindings,1]]
vault/dendron.topic.publishing-legacy.commands.md:10:![[dendron.topic.commands#publishing,1:#Dev]]
vault/dendron.release.changelog.md:346:You-can-now-do-references-to-notes-using-`![[note`-instead-of-`![[note]]`-syntax.-To-reference-a-header,-use-`![[note#foo`.-This-changes-makes-our-note-ref-syntax-more-consistent-with-our-wiki-link-syntax.-
vault/dendron.release.2020-11-29.md:51:![[dendron.release.changelog#0.17.2:#0.16.3]]
vault/dendron.workflows.todos.md:61:![[dendron.guides.tips#Moving-lines,1:#*]]
vault/dendron.release.2020-11-22.md:69:![[dendron.release.changelog#0.16.3:#0.16.1]]
vault/dendron.topic.tags.md:32:![[dendron.guides.tips#stylized-tags-using-custom-css:#*]]
vault/dendron.release.2020-12-21.md:50:![[dendron.release.changelog#0.20.2:#0.19.3]]
vault/dendron.topic.frontmatter.md:67:![[dendron.topic.publishing.configuration#Note-Configuration]]
vault/dendron.roadmap.project.n.2020.multi-vault.md:46:![[dendron.roadmap.project.n.2020.multi-vault#file-layout-for-legacy-workspace,1:#*]]
vault/dendron.roadmap.project.n.2020.multi-vault.md:50:![[dendron.roadmap.project.n.2020.multi-vault#file-layout-for-multi-vault-workspace,1:#*]]
vault/dendron.roadmap.project.n.2020.multi-vault.md:88:![[dendron.roadmap.project.n.2020.multi-vault#file-layout-for-multi-vault-workspace,1:#*]]
vault/dendron.roadmap.project.n.2020.multi-vault.md:123:![[dendron.roadmap.project.n.2020.multi-vault#file-layout-for-multi-vault-workspace,1:#*]]
vault/dendron.roadmap.project.n.2020.multi-vault.md:139:![[dendron.roadmap.project.n.2020.multi-vault#file-layout-for-multi-vault-workspace,1:#*]]
vault/dendron.release.2020-09-13.md:19:![[dendron.topic.refs#-references,1:#-*]]-
vault/dendron.release.2020-09-13.md:33:![[dendron.topic.cli#install]]
vault/dendron.release.2020-09-13.md:55:![[dendron.topic.commands#configure-pod,1:#*]]
vault/dendron.release.2020-09-13.md:71:![[dendron.topic.publishing.configuration#useprettyrefs,1:#*]]
vault/dendron.release.2020-09-13.md:75:![[dendron.topic.publishing.configuration#noindex,1:#*]]
vault/dendron.release.2020-09-13.md:90:![[dendron.topic.refs#wildcard-header-reference,1:#*]]
vault/dendron.release.2020-09-13.md:105:![[dendron.pro.dendron-snippet-maker#dendron-snippet-maker,1]]
vault/dendron.workflows.cache.md:34:![[dendron.topic.publishing.advanced#exclude-from-indexing,1:#*]]
vault/dendron.topic.pod.builtin.json.md:170:"body":"\n\n#-Inner-Note\n\n![[bar.two.child]]\n![[bar.two.child]]",
vault/dendron.release.2020-11-15.md:55:![[dendron.release.changelog#0.15.3:#0.14.2]]
vault/dendron.release.2020-12-14.md:58:![[dendron.release.changelog#0.19.3:#0.18.2]]
vault/dendron.dev.md:14:![[dendron.dev.setup#getting-started]]
vault/dendron.release.2021-01-04.md:61:![[dendron.release.changelog#0.22.2:#0.21.2]]
vault/dendron.release.2020-12-07.md:52:![[dendron.release.changelog#0.18.2:#0.17.2]]
vault/dendron.release.2020-11-08.md:24:![[dendron.community.roles#dendrologist,1:#*]]
vault/dendron.release.2020-11-08.md:48:![[dendron.release.changelog#changelog,1:#13.6]]
vault/dendron.dev.api.seeds.md:9:![[dendron.topic.seeds#seeds-🚧,1:#*]]
vault/dendron.topic.hierarchies.md:43:![[dendron.ref.layout#File-Tree,1:#Additional-Folders]]
vault/dendron.dev.design.engine.md:279:![[dendron.topic.refs#references,1:#*]]
vault/dendron.topic.pod.markdown.md:65:![[pro.dendron.feat.publish]]
vault/dendron.topic.pod.markdown.md:129:![[dendron.topic.refs#references,1:#*]]
vault/dendron.guides.cook.md:172:![[dendron.topic.commands#initialize-workspace:#*]]
vault/dendron.guides.cook.md:226:![[dendron.topic.workbench#outline-view,1:#*]]
vault/dendron.guides.cook.md:229:![[dendron.topic.preview#table-of-contents,1]]
vault/dendron.guides.cook.md:253:![[dendron.topic.cli#installation,1:#*]]
vault/dendron.workflows.bullet-journal.md:20:![[dendron.topic.special-notes#daily-journal,1:#*]]
vault/dendron.workflows.bullet-journal.md:24:![[dendron.topic.commands#go-next-sibling,1:#*]]
vault/dendron.workflows.bullet-journal.md:28:![[dendron.topic.schema#template,1:#*]]
vault/dendron.workflows.bullet-journal.md:34:![[dendron.guides.tips#moving-panes,1]]
vault/dendron.topic.capabilities.snapshot.md:14:![[dendron.topic.commands#snapshot-vault:#*]]
vault/dendron.topic.capabilities.snapshot.md:18:![[dendron.topic.commands#restore-vault:#*]]
vault/dendron.topic.snippets.md:23:![[dendron.pro.dendron-snippet-maker]]
vault/dendron.topic.refs.md:26:NOTE:-Dendron-recently-switched-the-note-ref-syntax-for-`![[foo]]`-to-`![[foo`.-The-videos-have-not-yet-been-updated-to-reflect-this-change
vault/dendron.release.2020-11-01.md:79:![[dendron.release.changelog#13.6:#0.12.11]]
vault/dendron.topic.seeds.md:28:![[dendron.topic.cli#installation,1:#*]]
vault/dendron.md:49:![[dendron.guides.install#Getting-Started,1:#Install]]
vault/dendron.release.2020-09-20.md:43:![[dendron.topic.seeds#seeds-🚧,1:#*]]
vault/dendron.release.2020-09-20.md:60:![[dendron.topic.lookup#creating-notes-with-a-selection,1:#*]]
vault/dendron.release.2020-09-20.md:70:![[dendron.topic.pod.builtin.json]]
vault/dendron.release.2020-09-20.md:86:![[dendron.topic.commands#publish,1:#*]]
vault/dendron.release.2020-09-20.md:90:![[dendron.topic.commands#copy-note-url,1:#*]]
vault/dendron.release.2020-10-19.md:43:![[dendron.community.showcase#open-pkm-catalogue,1:#*]]
vault/dendron.release.2020-10-19.md:44:![[dendron.community.showcase#open-aws-catalogue,1:#*]]
vault/dendron.release.2020-10-19.md:66:![[dendron.release.changelog#0.12.11,1]]
vault/dendron.pro.dendron-next-server.md:26:![[dendron.topic.cli#installation,1:#*]]
vault/dendron.workflows.academics.md:13:![[dendron.topic.markdown#footnotes,1:#*]]
vault/dendron.workflows.academics.md:17:![[dendron.topic.links#other-links,1]]
vault/dendron.guides.install.md:20:![[dendron.guides.cook#launch-the-command-bar,3:#*]]
vault/dendron.release.changelog.0-1-x.md:24:![[dendron.topic.seed-bank]]
vault/dendron.release.changelog.0-1-x.md:50:![[dendron.topic.multi-vault#specify-vault-location-when-creating-a-note,1:#*]]
vault/dendron.release.changelog.0-1-x.md:143:![[dendron.topic.commands#configure,1:#*]]
vault/dendron.release.changelog.0-1-x.md:226:![[dendron.topic.commands#-copy-note-link:#-*]]
vault/dendron.release.changelog.0-1-x.md:298:![[dendron.roadmap.project.n.2020.multi-vault#use-cases,1:#*]]
vault/dendron.release.changelog.0-1-x.md:348:![[dendron.topic.refs#wildcard-note-refs,1]]
vault/dendron.release.changelog.0-1-x.md:375:![[dendron.topic.cli#launchengineserver:#*]]
vault/dendron.release.changelog.0-1-x.md:395:![[dendron.topic.lookup.modifiers#copywikilink,1:#*]]
vault/dendron.release.changelog.0-1-x.md:401:![[dendron.topic.lookup.modifiers#multiselect,1:#*]]
vault/dendron.release.changelog.0-1-x.md:457:![[dendron.topic.commands#contribute:#*]]
vault/dendron.release.changelog.0-1-x.md:463:![[dendron.topic.capabilities.snapshot#Snapshots:#*]]
vault/dendron.release.changelog.0-1-x.md:524:![[dendron.topic.lookup#filter-toggle,1:#*]]
vault/dendron.release.changelog.0-1-x.md:527:![[dendron.topic.lookup.modifiers#filtertype:#*]]
vault/dendron.release.changelog.0-1-x.md:532:![[dendron.topic.publishing.configuration#config:#*]]
vault/dendron.release.changelog.0-1-x.md:536:![[dendron.topic.schema#schema-templates,1:#*]]
vault/dendron.release.changelog.0-1-x.md:565:![[dendron.topic.pod#publish,1:#*]]
vault/dendron.release.changelog.0-1-x.md:592:![[dendron.topic.pod#pods:#*]]
vault/dendron.release.changelog.0-1-x.md:602:![[dendron.topic.publishing.configuration#toc:#*]]
vault/dendron.release.changelog.0-1-x.md:626:![[dendron.topic.refs#recursive-reference,1]]
vault/dendron.release.changelog.0-1-x.md:683:![[dendron.guides.troubleshooting#error-upgrading:#*]]
vault/dendron.release.changelog.0-1-x.md:713:![[dendron.topic.extensions#useful-extensions]]
vault/dendron.release.changelog.0-1-x.md:715:![[dendron.guides.tips#copy-and-paste-web-content-into-dendron:#*]]
vault/dendron.release.changelog.0-1-x.md:717:![[dendron.guides.cook#mobile-support:#*]]
vault/dendron.release.changelog.0-1-x.md:780:![[dendron.roadmap.project.n.2020.server-migration#tasks,1:#*]]
vault/dendron.release.changelog.0-1-x.md:795:![[dendron.topic.snippets#default-snippets,1:#*]]
vault/dendron.release.changelog.0-1-x.md:799:![[dendron.pro.dendron-jekyll.topic.math#math,1:#*]]
vault/dendron.release.changelog.0-1-x.md:803:![[dendron.topic.publishing.configuration#siterepodir,1:#*]]
vault/dendron.release.changelog.0-1-x.md:820:![[dendron.roadmap.project.n.2020.server-migration#tasks,1:#*]]
vault/dendron.release.changelog.0-1-x.md:861:![[dendron.topic.lookup#split-toggle]]
vault/dendron.release.changelog.0-1-x.md:867:![[dendron.topic.commands#copy-note-ref:#*]]
vault/dendron.release.changelog.0-1-x.md:873:![[dendron.roadmap]]
vault/dendron.release.changelog.0-1-x.md:877:![[dendron.topic.lookup#lookup-menu:#*]]
vault/dendron.release.changelog.0-1-x.md:882:![[dendron.community.showcase#luke's-second-brain]]
vault/dendron.pro.dendron-jekyll.topic.frontmatter.md:46:![[dendron.topic.publishing.configuration#config:#*]]
data/dendron-config.json:116:"docs":"![[dendron.topic.lookup#notes,1:#schemas]]",
data/dendron-config.json:134:"docs":"![[dendron.topic.lookup#notes,1:#schemas]]",
data/dendron-config.json:153:"docs":"![[dendron.topic.lookup#notes,1:#schemas]]",
data/v0.10.7.yml:51:![[dendron.topic.cli#install]]
data/v0.10.7.yml:84:![[dendron.topic.commands#new-note-from-selection,1:#*]]
data/v0.10.7.yml:93:![[dendron.topic.commands#configure-pod,1:#*]]
data/v0.10.7.yml:110:![[dendron.topic.pod#export-config,1]]
data/v0.10.7.yml:135:![[dendron.topic.publishing#note-references,1:#*]]
data/v0.10.7.yml:144:![[dendron.topic.publishing#exclude-from-indexing,1:#*]]
data/v0.10.7.yml:209:![[dendron.topic.refs#wildcard-header-ref,1]]
data/v0.10.7.yml:218:![[dendron.pro.dendron-snippet-maker#dendron-snippet-maker,1]]
data/v0.11.9.yml:40:![[dendron.topic.seeds#seeds-🚧,1:#*]]
data/v0.11.9.yml:65:![[dendron.topic.lookup#creating-notes-with-a-selection,1:#*]]
data/v0.11.9.yml:82:![[dendron.topic.pod#json-pod,1:#Custom-Pods]]
data/v0.11.9.yml:123:![[dendron.topic.commands#publish,1:#*]]
data/v0.11.9.yml:188:![[dendron.topic.commands#copy-note-url,1:#*]]
data/v0.9.7.yml:102:![[demo.embed.block#head1,1]]
```

**Comment by kevinslin** [comment]
yeah, i’ve been meaning on adding a doctor command to fix it and have been
delaying on fixing this until that was in place 😅
